init.d.in

```diff
@@ -38,11 +39,15 @@ stop() {
   [ "$EUID" != "0" ] && exit 4
   if [ "$pid" != "" ] && ps -p $pid > /dev/null; then
     echo -n $"Shutting down $prog: "
-    kill $(fuser -n tcp $PORT)
-    exitcode=$?
-    echo
-    [ $exitcode -eq 0 ] && rm -f $lockfile
-    return $exitcode
+    pid=$(fuser -n tcp $PORT)
```

Comment:
of interest is that there is a `/v1/ping` endpoint that might be worth checking in addition to the port.  This will tell us if the program is actually responding to requests in addition to listening to the port.  Not related to this PR, but just noticed it.